src/main/resources/assets/controlify/lang/fr_fr.json

```diff
@@ -0,0 +1,393 @@
+  "controlify.controller_submission.message": "Veuillez soumettre certaines informations sur votre contrôleur à la base de données de Controlify pour l'ajouter dans une future mise à jour.\n\nControlify envoie les informations suivantes :\n- Les ID de fournisseur et de produit de votre contrôleur\n- Le GUID de votre contrôleur\n- Le nom de votre contrôleur (dans la zone ci-dessous)\n- La version de Controlify que vous utilisez actuellement\n\nCeci est complètement anonyme et ne stocke aucune de vos informations personnelles ou de compte.",
+  "controlify.controller_submission.operational_checkbox": "Votre contrôleur fonctionne-t-il avec Controlify ?",
+  "controlify.controller_submission.submit": "Soumettre les données",
+  "controlify.controller_submission.skip": "Passer",
+  "controlify.controller_submission.name_hint": "Nom du contrôleur",
+  "controlify.controller_submission.name_narration": "Champ de saisie du nom du contrôleur",
+  "controlify.controller_submission.invalid_name": "Nom de contrôleur non valide. Utilisez uniquement des lettres, des chiffres et des espaces."
+
+  "controlify.error.hid": "Détection du contrôleur désactivée",
+  "controlify.error.hid.desc": "L'identification du contrôleur a échoué, donc toutes les modifications de configuration du contrôleur ne seront pas conservées. Veuillez consulter les logs pour plus d'informations."
+
+  "controlify.battery_level.unknown": "Inconnu",
+  "controlify.battery_level.empty": "Vide",
+  "controlify.battery_level.low": "Faible",
+  "controlify.battery_level.medium": "Moyen",
+  "controlify.battery_level.high": "Élevé",
+  "controlify.battery_level.full": "Plein"
+
+  "controlify.command.vibratecontroller.static.single": "Contrôleur vibré d'1 joueur.",
+  "controlify.command.vibratecontroller.static.multiple": "Contrôleurs vibrés de %s joueurs.",
+  "controlify.command.vibratecontroller.pos.single": "Contrôleur vibré d'1 joueur à %s, %s, %s.",
+  "controlify.command.vibratecontroller.pos.multiple": "Contrôleurs vibrés de %s joueurs à %s, %s, %s.",
+  "controlify.command.vibratecontroller.entity.single": "Contrôleur vibré d'1 joueur depuis la position de %s.",
+  "controlify.command.vibratecontroller.entity.multiple": "Contrôleurs vibrés de %s joueurs depuis la position de %s."
+
+  "controlify.compat.svc.hold": "(Maintenir)",
+  "controlify.compat.svc.toggle": "(Basculer)",
+
+  "controlify.hat_state.up": "Haut",
+  "controlify.hat_state.down": "Bas",
+  "controlify.hat_state.left": "Gauche",
+  "controlify.hat_state.right": "Droite",
+
+  "controlify.joystick_mapping.unmapped.axis": "Axe #%s",
+  "controlify.joystick_mapping.unmapped.button": "Bouton #%s",
+  "controlify.joystick_mapping.unmapped.hat": "#%s",
+  "controlify.joystick_mapping.unmapped.axis_direction.negative": "(Négatif)",
+  "controlify.joystick_mapping.unmapped.axis_direction.positive": "(Positif)"
+
+
+
+  "controlify.joystick_mapping.tm_t16000.axis.roll": "Roll",
```

Comment:
"controlify.joystick_mapping.tm_t16000.axis.roll": "Roulis",
  "controlify.joystick_mapping.tm_t16000.axis.pitch": "Tangage",
  "controlify.joystick_mapping.tm_t16000.axis.yaw": "Lacet",
  "controlify.joystick_mapping.tm_t16000.axis.throttle": "Manette des gaz",
  "controlify.joystick_mapping.tm_t16000.button.trigger": "Gâchette",

Reply:
I'm not sure about thoses. I trust you.

Modified.